blogposts/2020/3.18/announcing-sourcegraph-3.18.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Sourcegraph 3.18: Search across multiple revisions and non-master branches, custom search pages, campaigns now support GitLab, new C++ precise code intelligence"
+author: Laureen Hudson
+publishDate: 2020-MM-20T10:00-07:00
+tags: [blog]
+slug: sourcegraph-3.18
+heroImage: /blog/3.18-release-blog-img.jpg
+published: true
+---
+
+We’re improving Sourcegraph universal code search continuously, to help you solve the Big Code problems you face, all day every day. Here's how we're doing it.
+
+-   [Indexed non-master branches](https://about.sourcegraph.com/blog/indexed-non-master-branches)
+
+    Often, you need to search code that isn’t in your default branch, like long-lived release branches or important tags. In Sourcegraph 3.18, we now use an index on the default branch to speed up search operations, with the added capability to index branches other than the default branch.
+
+-   [Search across multiple revisions](https://about.sourcegraph.com/blog/search-multiple-revisions)
+
+    You can now search across multiple revisions of the same repository by listing multiple branch names (or other revision specifications) separated by `:` in your query. This helps you understand the differences between code at different branches (especially for release branches that have diverged).
```

Comment:
Instead of saying `separated by : in your query`, which I don't think people will understand ("where do I put the `:`?"), I would recommend just giving an example.

As for the canonical example of when this is useful, IMO it is "See all release branches that contain a particular bug so you can be sure to fix it in all the right places", not the example listed.

Reply:
You can now search across multiple revisions of the same repository by listing multiple branch names (or other revision specifications). For example, to search across multiple specific branches, you’d use something like `repo:myrepo@branch1:branch2:branch2` in your query. This helps you see all release branches that contain a particular bug so you can be sure to fix it in all the right places.

Reply:
done

Reply:
done